curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+WordPress
 
 ---
 
-Placeholder distractor 2
+Sketch
 
 ---
 
-Placeholder distractor 3
+Excel
 
 #### --answer--
 
-Placeholder answer
+Paletton
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which color is considered low contrast with white?
 
 #### --distractors--
 
-Placeholder distractor 1
+Black
 
 ---
 
-Placeholder distractor 2
+Dark Blue
 
 ---
 
-Placeholder distractor 3
+Dark Green
 
 #### --answer--
 
-Placeholder answer
+Light Gray
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool allows you to test CSS live on a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+CodePen
 
 ---
 
-Placeholder distractor 2
+JSFiddle
 
 ---
-
-Placeholder distractor 3
+Live CSS Editor
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property makes an element invisible but still takes up space?
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+z-index: -1;
 
 ---
-
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+visibility: hidden;
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for graphics?
 
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+7:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which feature of Chrome helps in accessibility audits?
 
 #### --distractors--
-
-Placeholder distractor 1
+DevTools
 
 ---
 
-Placeholder distractor 2
+Extensions
 
 ---
 
-Placeholder distractor 3
+Bookmark Manager
 
 #### --answer--
 
-Placeholder answer
+Lighthouse
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a reason to hide content on a webpage?
 
 #### --distractors--
 
-Placeholder distractor 1
+To improve SEO
 
 ---
 
-Placeholder distractor 2
+For user experience
 
 ---
 
-Placeholder distractor 3
+To save bandwidth
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which color combination is generally recommended for accessibility?
```

Comment:
I would probably change this to something like "Which color combination has the potential to provide the highest contrast ratio", since pure black on white will give you the highest possible ratio. But since there are so many different shades of colors, it is possible to meet minimum contrast ratios with green on red or blue on yellow.